legal-api/report-templates/template-parts/registration/completingParty.html

```diff
@@ -12,10 +12,19 @@
                         {% if party.officer.partyType == 'organization' %}
                            <span class="capitalize-text">{{ party.officer.organizationName }}</span>
                         {% else %}
-                           <span class="capitalize-text">{{ party.officer.lastName }}</span>,
-                           <span class="capitalize-text">{{ party.officer.firstName }}</span>
-                           {% if party.officer.middleName %}
-                              <span class="capitalize-text">{{ party.officer.middleName }}</span>
+                           {% if party.officer.firstName %}
+                              <span class="capitalize-text">{{ party.officer.lastName }}</span>,
+                              <span class="capitalize-text">{{ party.officer.firstName }}</span>
+                              {% if party.officer.middleName %}
+                                 <span class="capitalize-text">{{ party.officer.middleName }}</span>
+                              {% endif %}
+                           {% else %}
+                              {% if party.officer.middleName %}
+                                 <span class="capitalize-text">{{ party.officer.lastName }}</span>,
+                                 <span class="capitalize-text">{{ party.officer.middleName }}</span>
+                              {% else %}
+                                 <span class="capitalize-text">{{ party.officer.lastName }}</span>
+                              {% endif %}
```

Comment:
I think you might be able to simplify this template by using [_get_party_name](https://github.com/bcgov/lear/blob/6e4130fda5c79c06ec576f5911e92dbb697eacb3/legal-api/src/legal_api/reports/report.py#L707-L717) in [_format_registration_data](https://github.com/bcgov/lear/blob/6e4130fda5c79c06ec576f5911e92dbb697eacb3/legal-api/src/legal_api/reports/report.py#L260-L259).  

Basically, loop through each party in the filing and just add a new field to the party to represent the display name for the party.  It won't be the exact code but you can use this piece of [code](https://github.com/bcgov/lear/blob/6e4130fda5c79c06ec576f5911e92dbb697eacb3/legal-api/src/legal_api/reports/report.py#L847-L846) as a reference too.

With this, I'm thinking you will only need to use something like `<span class="capitalize-text">{{ party.officer.fullName }}</span>`.  And it will also work for the case where partyType == 'organization' I'm thinking.

Reply:
I pushed a new comment with adding fullName field, not sure whether the new function name are standards